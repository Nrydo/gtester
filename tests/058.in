8652647
986021379
220637985