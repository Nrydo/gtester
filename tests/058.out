9287813103125686